tests/test_cli_bitcoind.py

```diff
@@ -27,12 +28,15 @@ def test_bitcoind(caplog):
 
 
 def test_elements(caplog):
-    # caplog.set_level(logging.DEBUG)
+    caplog.set_level(logging.DEBUG)
 
     runner = CliRunner()
     result = runner.invoke(elementsd, ["--no-mining", "--cleanuphard"])
     print(result.output)
     if result.exception != None:
+        if "Couldn't find executable elementsd" in str(result.exception):
```

Comment:
Well, that's a good point. In the mid-term, i think we should somehow create different labels for the tests and make it possible that people skip certain test-types. Did that e.g. here:
https://github.com/cryptoadvance/specter-desktop/blob/master/tests/test_wallet_manager.py#L13

Existing markers are defined in pytest-ini
https://github.com/cryptoadvance/specter-desktop/blob/master/pytest.ini#L5

You could then run the tests without the slow ones like this:
```
pytest -m "not slow"
```
Haven't figured out how devs could set their own default settings which would be a good thing to have, though.
What do you think about that approach?
And yes, i guess i need to do some better documentation about:
```
./tests/install_bitcoind.sh --elements compile
```

Reply:
The test suite runs in under 5 minutes on my old Macbook Air (that's with the two elements tests skipping themselves). So I'm not worried yet about getting finer control over the tests. I think the bigger issue is that we just need more tests! Stepan caught at least one bug in my code that should have ideally been caught by a test case.

Reply:
Yeah, more tests are always better. 5 mins is to me already too long. Also it's the distribution. On my nuc:
* pytest -m "not slow" --> 81 tests in 10s
* pytest --> 89 tests in 3:17 mins